### DISCUSSION TITLE
How to redirect when navigating to non existent route

### DISCUSSION THREAD

**Discussion by joshiain** [sveltejs/kit#6011] [Q&A]
Previously I would export a load function inside `__error.svelte` which would contain a redirect back to `/`.

With the latest changes, there is no longer the ability to use a load function with `+error.svelte`. 

How might I now implement a redirect back to `/` when a user visits a non existent route?

**Comment by ssbm-oro** [comment]
Can you use [goto('/')](https://kit.svelte.dev/docs/modules#$app-navigation-goto) in `+error.svelte`?

**Reply by joshiain** [reply]
As `goto` doesn't work on the server, this only works if you wrap it in a browser check guard like so:

```javascript
$: if (browser) {
    goto('/');
}
```

Seems good thanks! Wonder if there's any drawbacks to this

**Comment by gregg-cbs** [comment]
Is this the official answer to this problem?
